include/cuco/detail/prime.hpp

```diff
@@ -25,7 +25,7 @@
 namespace cuco {
 namespace detail {
 
-inline constexpr std::array<int64_t, 140746> primes = {
+inline constexpr std::array<int64_t, 140739> primes = {
```

Comment:
```suggestion
inline constexpr std::array<uint64_t, 140739> primes = {
```
I recall you changed the data type to `uint64_t` in `refactor` branch.

Reply:
Yes, the primary reason for the change was that I needed this for the new `next_prime` function in the `refactor` branch to be able to support the case where `cuco::extent::size_type == size_t`. I does not affect the `dev` branch so I thought we leave it for now.